tests/integration/targets/helm/tasks/test_helm_take_ownership.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Test helm take ownership
+  vars:
+    helm_namespace: "{{ test_namespace[12] }}"
+  block:
+
+    - name: Initial chart installation (no flag set)
+      helm:
+        binary_path: "{{ helm_binary }}"
+        chart_ref: "{{ chart_test_oci }}"
+        release_name: test-take-ownership
+        release_namespace: "{{ helm_namespace }}"
+        create_namespace: true
+      register: install
+
+    - name: Validate that take-ownership flag is not set
+      assert:
+        that:
+          - install is changed
+          - '"--take-ownership" not in install.command'
+
+    - name: Initial chart installation (take-onwership flag set)
+      helm:
+        binary_path: "{{ helm_binary }}"
+        chart_ref: "{{ chart_test_oci }}"
+        release_name: test-take-ownership
+        release_namespace: "{{ helm_namespace }}"
+        take_ownership: true
+      register: install
+      ignore_errors: true
+
+    - name: Validate that take-ownership flag IS set if helm version is >= 3.17.0
+      assert:
+        that:
+          - install is changed
+          - '"--take-ownership" in install.command'
+      when: '"v3.17.0" <= helm_version'
+
+    - name: Validate that feature fails for helm < 3.17.0
+      assert:
+        that:
+          - install is failed
+          - '"-take_ownership requires helm >= 3.17.0" in install.msg'
+      when: 'helm_version < "v3.17.0"'
+
+  always:
+    - name: Remove helm namespace
```

Comment:
If the same namespace is reused in other tests, a race condition may cause a failure. Please ensure that `{{ test_namespace[12] }}` is unique